Assets/Scripts/GameplayLayer/Intruder/Chicken.cs

```diff
@@ -38,7 +33,7 @@ public override void Intrude()
             if(_currentTime > _timer)
             {
                 //mengacaukan tempat sampah (animasi tempat sampah berantakan)
-                Debug.Log("Kurangi poin player"); //publish event to decrease score
+                PublishSubscribe.Instance.Publish<MessageRemoveScore>(new MessageRemoveScore(_amountScoreDecrease));
```

Comment:
```suggestion
                PublishSubscribe.Instance.Publish<MessageRemoveScore>(new MessageRemoveScore("Normal"));
```
Untuk ngatur score kuranginnya berapa nanti di script ScoreController aja biar sekalian satu tempat.

Reply:
Ohh okee, aku gatau tadi parameternya ada apa aja. Hapus aja berarti variabel amountScoreDecrease